.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+-   repo: git://github.com/pre-commit/mirrors-puppet-lint
```

Comment:
This is a file to use pre-commit.com at development time, pretty useful for githooks management